JavaScript/JavaScriptSDK/snippet.js

```diff
@@ -32,6 +32,7 @@
             var originalArguments = arguments;
             // Queue-up a call to the real method
             appInsights.queue.push(function() {
+                
```

Comment:
Please also record the original call time and add it to argument list of `startTrackPage` and `stopTrackPage` methods.